synth.metadata

```diff
@@ -3,23 +3,16 @@
     {
       "git": {
         "name": ".",
-        "remote": "https://github.com/googleapis/nodejs-retail.git",
-        "sha": "9ea6f9b76989c0b6d15c4cdf9dcf115a97829153"
+        "remote": "git@github.com:googleapis/nodejs-retail.git",
```

Comment:
IIRC this will generate an extra synth PR. Maybe revert back to `https://` as synthtool does?